pages/platform/databases/connect-kubernetes-to-managed-mysql/guide.en-asia.md

```diff
@@ -0,0 +1,302 @@
+
+You need to have [Helm](https://docs.helm.sh/){.external} installer on your workstation and your cluster, please refer to the [How to install Helm on OVHcloud Managed Kubernetes Service](../../../kubernetes/installing-helm/) tutorial.
+
+Finally, you need to be able to order a database solution in the OVHcloud Control Panel, as explained at [Getting started with Public Cloud Databases](../getting-started/).
+
+## Create a MySQL database
+
+### Subscribing to the service
+
+Log in to your [OVHcloud Control Panel](https://ca.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/asia/&ovhSubsidiary=asia) and switch to `Public Cloud`{.action} in the top navigation bar. After selecting your Public Cloud project, click on `Databases`{.action} in the left-hand navigation bar under **Storage**.
+
+Click on the button `Create a database instance`{.action}. (`Create a service`{.action} if your project already contains databases.)
+
+### Step 1: Select MySQL database
+
+Click on the MySQL database and then select the version to install from the drop-down menu. Click on `Next`{.action} to continue.
+
+![Choose MySQL database](images/connect-kubernetes-to-managed-mysql01.png){.thumbnail}
+
+### Step 2: Select a solution
+
+In this step, choose an appropriate service plan. You will be able to upgrade the plan after creation.
+
+![Choose plan](images/connect-kubernetes-to-managed-mysql02.png){.thumbnail}
+
+Please visit the [capabilities page](../mysql/capabilities) for detailed information on each plan's properties.
+
+Click on `Next`{.action} to continue.
+
+### Step 3: Select a location
+
+Choose the geographical region of the data centre in which your database will be hosted.
+
+![Choose region](images/connect-kubernetes-to-managed-mysql03.png){.thumbnail}
+
+Click on `Next`{.action} to continue.
+
+### Step 4: Configure database nodes
+
+You can choose the node model in this step. The initial and maximum numbers of nodes depends on the solution chosen in step 2.
+
+![Order nodes](images/connect-kubernetes-to-managed-mysql04.png){.thumbnail}
+
+Please visit the [capabilities page](../mysql/capabilities/) for detailed information on the hardware resources and other properties of the MySQL database installation.
+
+Take note of the pricing information and click on `Next`{.action} to continue.
+
+### Step 5: Configure your options
+
+You can name your database in this step.
+
+![Configure options](images/connect-kubernetes-to-managed-mysql05.png){.thumbnail}
+
+### Connecting a private network (optional)
+
+For now, the solution is not compatible with private networks.
+
+### Step 6: Summary and confirmation
+
+The final section will display a summary of your order as well as the API equivalent of creating this database instance with the [OVHcloud API](../../../api/first-steps-with-ovh-api/).
```

Comment:
suggestion:
"The last section will show a summary of your order as well as the API equivalent of creating this database instance with the [OVHcloud API] (../../../ api / first-steps-with- ovh-api /)."